.travis.yml

```diff
@@ -26,7 +26,7 @@ before_script:
 
 # command to run tests, e.g. python setup.py test
 script:
-  - tox -e $TOX_ENV
+  - tox -e $TOX_ENV -r
```

Comment:
Though at least in one instance locally, tox failed to install new requirements when I added them to the req's file. So probably it is necessary.